docs/source/api-reference/authentication.md

```diff
@@ -0,0 +1,168 @@
+# Authentication
+
+Nyrkiö uses JWT (JSON Web Token) authentication for API access.
```

Comment:
Yeah but we shouldn't really... Or I mean the format is good, but we need to store logged in users in the DB so that we can also log them out. Formal JWT is only good for short lived sessions like 10 minutes or 10 hours max